Sources/PostgresNIO/Data/PostgresData+Range.swift

```diff
@@ -0,0 +1,133 @@
+import NIOCore
+
+// MARK: PostgresRange
+
+public struct PostgresRange<B> {
```

Comment:
we should make this `@usableFromInline` instead of public. This way it is easier for us to change, if new requirements appear when we add conformance for dates and other stuff.

Reply:
See new commit https://github.com/vapor/postgres-nio/pull/330/commits/23cd10f201ff3f10344eabc25c455cdbb2111a3e